src/deadline/cinema4d_submitter/font_utils.py

```diff
@@ -24,62 +22,83 @@
 logger = logging.getLogger(__name__)
 
 
-def get_font_metadata(font_path: str) -> Optional[Dict[str, Any]]:
+@dataclass
+class FontMetadata:
+    """
+    Dataclass representing font metadata extracted from a font file.
+    """
+
+    file_path: str
+    family_name: Optional[str] = None
+    style: Optional[str] = None
+    full_name: Optional[str] = None
+    postscript_name: Optional[str] = None
+    raw_names: Optional[Dict[int, str]] = None
+
+    def __post_init__(self):
+        """Initialize raw_names as empty dict if None."""
+        if self.raw_names is None:
+            self.raw_names = {}
+
+
+def get_font_metadata(font_path: str) -> Optional[FontMetadata]:
     """
     Extract font metadata from a font file using fontTools.
 
     Args:
         font_path (str): Path to the font file
 
     Returns:
-        Optional[Dict[str, Any]]: Font metadata dictionary or None if extraction fails
+        Optional[FontMetadata]: Font metadata dataclass or None if extraction fails
     """
 
     if not font_path or not os.path.isfile(font_path):
         return None
 
+    if not is_windows():
+        return None
+
+    # Import fontTools only when needed and on Windows
+    try:
+        from fontTools import ttLib
+    except ImportError:
+        return None
+
     try:
         font = ttLib.TTFont(font_path)
         names_table = font["name"].names
 
-        # Extract key font information
-        metadata: dict[str, Any] = {
-            "file_path": font_path,
-            "family_name": None,
-            "style": None,
-            "full_name": None,
-            "postscript_name": None,
-            "raw_names": {},
-        }
+        # Create FontMetadata instance
+        metadata = FontMetadata(file_path=font_path)
 
         # Build raw names table for debugging
         try:
-            metadata["raw_names"] = {i: str(names_table[i]) for i in range(len(names_table))}
+            metadata.raw_names = {i: str(names_table[i]) for i in range(len(names_table))}
```

Comment:
Is `dict[int, str]` the best type for this? Do we just need a list of raw names?